Controls/Metadata.xaml.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace DolphinDynamicInputTextureCreator.Controls
+{
+    /// <summary>
+    /// Interaktionslogik für metadata.xaml
```

Comment:
Super minor but can you change this to english? (or remove? I know it was auto generated).  I just want to avoid someone seeing this and thinking its ok to add comments in other languages.

Reply:
have changed it, I also did not notice.